fitbenchmarking/fitbenchmarking
# JOSS paper

#### Description of Work

Fixes #639 

Adding JOSS paper to repo

#### Testing Instructions

1. Check the paper for any typos/inconsistencies. 

Function: Does the change do what it's supposed to?

Tests: Does it pass? Is there adequate coverage for new code?

Style: Is the coding style consistent? Is anything overly confusing?

Documentation: Is there a suitable change to documentation for this change?
